src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -335,9 +336,16 @@ export function getPrismaClient(config: GetPrismaClientOptions): any {
           values: JSON.stringify(queryInstance.values),
           __prismaRawParamaters__: true,
         }
-      } else {
+      } else if ("string" === typeof stringOrTemplateStringsArray) {
         // Called with prisma.raw(string)
         query = stringOrTemplateStringsArray
+      } else {
+        // called with prisma.raw(prisma.sql\`\`)
+        query = stringOrTemplateStringsArray[sqlOutput]
+        parameters = {
+          values: JSON.stringify(stringOrTemplateStringsArray.values),
+          __prismaRawParamaters__: true,
```

Comment:
Typo here

Reply:
Hm, which part? :)

Reply:
Paramaters -> Parameters

Reply:
Ah, this is the actual name used in the codebase.

https://github.com/prisma/prisma/blob/788f9e5b1ca024a27d24a4eb8380eb87f7b474b6/src/packages/client/src/runtime/query.ts#L684

You want this handled in another PR or this one?

Reply:
I believe another PR would be better. Getting this one merged and documented would be a big win, I really need it... 
@timsuchanek @janpio

Reply:
Separate PR sounds better, indeed.